diesel_cli/README.md

```diff
@@ -131,7 +131,7 @@ Diesel can generate a bash completion script for itself:
 #### linux
 
 ```sh
-$ diesel bash-completion > /etc/bash_completion.d/diesel
+$ echo 'source <(diesel completions bash)' >> ~/.bashrc
```

Comment:
I've tried this command locally. It does not work for me, but this is probably related to my shell. We should probably just reword this section to just saying something like: 
Diesel can generate shell completion scripts via `diesel completions <shell>`. See the help message of this command for a list of supported shells. Checkout the documentation of your shell to see how you could use this completions.

Reply:
Good point. Another hint, this works with bash only and you have to start a new shell or run `source ~/.bashrc` in the current one.